agent/helpers.go

```diff
@@ -98,11 +97,12 @@ func (h Helper) isRouteExist(ip net.IP, netmask string) error {
 }
 
 // createRoute creates IP route, returns nil if success and error otherwise.
-func (h Helper) createRoute(ip net.IP, netmask string, via string, dest string) error {
+func (h Helper) createRoute(ip net.IP, netmask string, via string, dest string, extraArgs ...string) error {
```

Comment:
But 'via' just happens to be a keyword in ip, it doesn't have to be.

What I mean is: A route needs a couple of things to be defined:

* address range
* a device
* a source address

Those things are independent of the underlying route-setting code. If we name this parameter 'via' and this also just happens to be what's used in ip then it's really more a coincidence, isn't it?